### ISSUE TITLE
"Cannot copy a file over itself" dialog

### ISSUE THREAD

**Issue by BWPanda** [elementary/files#1359]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Describe the bug
I was copying a bunch of files from one directory to another, and I didn't realise that two of the files I was copying were actually symlinks to the directory I was copying them to. So I got this message popup:

![Screenshot from 2020-06-07 20-46-45](https://user-images.githubusercontent.com/2385329/83966906-723e4880-a900-11ea-8332-7ba2c0329b9b.png)

Ok, no worries, except that it doesn't tell me what file(s) it's referring to. I had to manually check the source and destination files to see what was different before I realised what the issue was.

A simple addition of the filename in question to this dialog would be a good idea.

## Platform Information
![](https://user-images.githubusercontent.com/2385329/83958077-10a2bd80-a8b1-11ea-8f63-2ce2c3cd6558.png)

- [ ] I'm using the latest version from git that I've manually compiled
- [x] I'm using the latest released stable version

**Comment by jeremypw** [comment]
Sounds reasonable.  Need to decide on suitable wording.

**Comment by mrbvrz** [comment]
Yup I seem to have the same issue, I think because of the latest `File` update, `Paste into a selected folder when using Ctrl + V`

**Comment by jeremypw** [comment]
The relevant code appears unchanged so this needs addressing.

**Comment by jeremypw** [comment]
This does not seem to be a problem as described in OS8.1 - it is valid to copy a link to the destination into the destination so the warning does not appear.  However it is still invalid to copy a child folder into itself.  The message now reads "The destination folder is inside the source folder".  It is still a reasonable request to include the actual file paths in the message though.
